packages/prop-house-webapp/src/utils/getImageFromDescription.ts

```diff
@@ -0,0 +1,33 @@
+import { StoredProposalWithVotes } from '@nouns/prop-house-wrapper/dist/builders';
+import isImage from 'is-image-url';
+
+const getImageFromDescription = async (proposal: StoredProposalWithVotes) => {
+  // proposal description field
+  const what = proposal.what;
```

Comment:
good idea to use "proposalDescription" here lol

Reply:
https://github.com/Prop-House/prop-house-monorepo/pull/407/commits/2b59d49574c42f1f0d7d2081e81753ebcee38cb8